api/services/auth.js

```diff
@@ -128,6 +129,68 @@ async function registration({ firstName, lastName, email, password, token }) {
   });
 }
 
+/**
+ * TODO: It needs to think about creating link for reset password
+ * @param {*} param0 - object which contents email field
+ */
+
+const resetPasswordRequest = async ({ email }) => {
+  try {
+    let v = await db.Users.findOne({ where: { email } });
+    let linkId = v && (await hashHelpers.createHash(email));
```

Comment:
You always check the `v` value. Why you can't check it in the start of the function like: 
`if(v) { 
  throw new Error('My error')
}`
Please check all your files and fix it